db/migrate/20150828153331_change_business_support_edition_areas_to_gss_codes.rb

```diff
@@ -0,0 +1,22 @@
+class ChangeBusinessSupportEditionAreasToGssCodes < Mongoid::Migration
+  def self.up
+    # Editing of an edition with an Archived artefact is not allowed
+    Edition.skip_callback(:save, :before, :check_for_archived_artefact)
+
+    BusinessSupportEdition.all.each do |edition|
+      edition.area_gss_codes = Area.areas_for_edition(edition).map { |area|
+        area.codes["gss"]
+      }.compact
+
+      edition.save!(validate: false) # Published editions can't be edited.
+    end
+  end
+
+  def self.down
+    # If this needs rolling back perminantly, the area_gss_codes field should
+    # probably just be removed from GOV.UK Content Models.
+    #
+    # .up is only adding data, not changing existing data, so I assume
+    # a temporary rollback doesn't really require any action.
```

Comment:
@jennyd, you'll have to show me how to do that in the MongoDB/Rails world. All I could think of was doing something like `edition.area_gss_codes = []` to empty the field